signer/tests/integration/bitcoin_validation.rs

```diff
@@ -252,22 +229,28 @@ async fn one_invalid_deposit_invalidates_tx() {
 
     let aggregate_key = setup.signers.signer.keypair.public_key().into();
 
-    let test_state = TestSignerState::with_defaults(chain_tip, aggregate_key);
+    let request = BitcoinPreSignRequest {
+        request_package: vec![TxRequestIds {
+            deposits: setup.deposit_outpoint_messages(),
+            withdrawals: Vec::new(),
+        }],
+        fee_rate: TEST_FEE_RATE,
+        last_fees: None,
+    };
 
     let btc_ctx = BitcoinTxContext {
         chain_tip: chain_tip_block.block_hash,
         chain_tip_height: chain_tip_block.block_height,
-        request_packages: vec![TxRequestIds {
-            deposits: setup.deposit_outpoint_messages(),
-            withdrawals: Vec::new(),
-        }],
         signer_public_key: setup.signers.keys[0],
         aggregate_key,
-        signer_state: test_state.get_btc_state(&ctx).await.unwrap(),
+        utxo: get_signer_utxo(&ctx, &chain_tip).await.unwrap(),
     };
 
-    let validation_data = btc_ctx.construct_package_sighashes(&ctx).await.unwrap();
-    // There are a few invariants that we uphold for our validation data.
+    let validation_data = request
+        .construct_package_sighashes(&ctx, &btc_ctx)
+        .await
+        .unwrap();
+    // There a re a few invariants that we uphold for our validation data.
```

Comment:
typo: `a re` -> `are`

Reply:
fixed in https://github.com/stacks-network/sbtc/pull/930/commits/70fb3ecea2e16530d22333f4f5ecb978f01b7a5d